### ISSUE TITLE
runtime alwas return "WARNING"

### ISSUE THREAD

**Issue by NeverUsedID** [BaldMansMojo/check_vmware_esx#83]
` ./check_vmware_esx --datacenter 192.168.0.1--sslport 443  --select runtime  --authfile vmware_credentials
`
returns 

```
WARNING: 0/43 VMs suspended - 20/43 VMs powered off - 32/43 VMs powered on - 3/3 Hosts powered on - 0/3 Hosts powered off/standby/unknown - No VMware Clusters - 43 VMs checked for VMWare Tools state, 0 are not OK.
 - 1/1 Vcenters green - 0/1 Vcenters red - 0/1 Vcenters yellow - 0/1 Vcenters gray - 0 config issues  - 0 config issues ignored


```
Why is this warning and not okay ?

thx for reply !

**Comment by BaldMansMojo** [comment]
Yep - thats right because "20/43 VMs powered off". You can map to OK with --ignore_warning. The warning was a behaviuor requested by users because normally VMs should not be down.

Regards - Martin

**Comment by NeverUsedID** [comment]
okay, thank you very much for help!
